packages/core/plugins/core-plugins-server-internal/BUILD.bazel

```diff
@@ -0,0 +1,162 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "core-plugins-server-internal"
+PKG_REQUIRE_NAME = "@kbn/core-plugins-server-internal"
+
+SOURCE_FILES = glob(
+  [
+    "**/*.ts",
+  ],
+  exclude = [
+    "**/*.config.js",
+    "**/*.test.*",
+    "**/*.stories.*",
+    "**/__snapshots__/**",
+    "**/integration_tests/**",
+    "**/mocks/**",
+    "**/scripts/**",
+    "**/storybook/**",
+    "**/test_fixtures/**",
+    "**/test_helpers/**",
+  ],
+)
+
+SRCS = SOURCE_FILES
+
+filegroup(
+  name = "srcs",
+  srcs = SRCS,
+)
+
+NPM_MODULE_EXTRA_FILES = [
+  "package.json",
+]
+
+RUNTIME_DEPS = [
+  "@npm//moment",
+  "@npm//rxjs",
+  "@npm//semver",
+  "@npm//type-detect",
+  "@npm//lodash",
+
```

Comment:
I'm not sure if leaving empty lines is an issue or not. It was easier to ensure all the correct dependencies are declared.